kubernetes/.openapi-generator/COMMIT

```diff
@@ -0,0 +1,2 @@
+Requested Commit: v3.3.4
+Actual Commit: 2353d71d4b02be6dbabe25aac1a9e56eb3b812a2
```

Comment:
Since its been a while can you bump this to `v4.0.3`, commit `203fc812bda4267020165e656f55185f72df4cc0`?

Reply:
Also just curious, how often would the version need to be bumped in the future?

Reply:
I believe that we rarely bump the version of swagger-codegen right now.